slack_sdk/socket_mode/websockets/__init__.py

```diff
@@ -65,6 +67,7 @@ def __init__(
         web_client: Optional[AsyncWebClient] = None,
         auto_reconnect_enabled: bool = True,
         ping_interval: float = 10,
+        trace_enabled: bool = False,
```

Comment:
As we added this flag to the parent class, this should be added to this websockets based implementation too.